packages/sdk/src/controllers/DocumentController.ts

```diff
@@ -50,4 +51,19 @@ export class DocumentController {
             .loadDocument(parsedDoc, JSON.stringify(options))
             .then((result) => getEditorResponseData<null>(result));
     };
+
+    /**
+     * This method will create a new document, based on a document preset
+     * @param preset the preset where we will create a document of.
+     * @returns the new document created and loaded inside of the canvas
+     */
+    createAndLoad = async (preset: LayoutPreset) => {
+        const res = await this.#editorAPI;
+
+        const parsedDoc = JSON.stringify(preset);
+
+        return res
+            .createAndLoadDocument(parsedDoc)
```

Comment:
So, will the engine be responsible for creating a `Document` based on the `LayoutPreset`?

Reply:
Jep, working on it